tests/test_models.py

```diff
@@ -456,6 +456,31 @@ def test_invalid_expressions_access(self, f_obj):
 
 
 class TestExpressions:
+    def test_bulk_update(self):
+        if VERSION[:2] < (2, 2):
```

Comment:
Usually, we skip tests via `pytest.mark.skipif` decorator - I'd prefer to have it here as well